fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/IDatabaseTranslator.java

```diff
@@ -145,4 +156,16 @@
      * @return
      */
     boolean clobSupportsInline();
+    
+    /**
+     * The main type of the database
+     * @return
+     */
+    DbType getType();
+    
+    /**
+     * The name of the "DUAL" table...that special table giving us one row/column.
+     * @return
+     */
+    String dualTableName();
```

Comment:
Where is this used? it's implemented in 3 spots, and not used.  Please add a backlog issue to implement something that uses it, or remove the unused code.

Reply:
Are you referring to getType or dualTableName? They are framework/utility methods. I don't think everything has to be used today if it might be useful in the future. It was in use at one point during the refactor, but if it's no longer used, I don't think we should remove it, only to have to implement it again in the future should it be needed.